### PR TITLE
Rename `s/analysis-dev/saveourtool/g` in URLs

### PR DESCRIPTION
**Pull request by unix-junkie**
### What's done:

 * URLs (incl. those from badges) have been corrected to accommodate for the organization name change.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ![Build and test](https://github.com/cqfn/diKTat/workflows/Build%20and%20test/badge.svg)
```
Comment (nulls) [inline README.md:3]: I guess these urls need to be updated too:
`https://github.com/cqfn/diKTat` to `https://github.com/saveourtool/diktat`

**Review by nulls — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-[![codecov](https://codecov.io/gh/analysis-dev/diKTat/branch/master/graph/badge.svg)](https://codecov.io/gh/analysis-dev/diKTat)
+![Build and test](https://github.com/saveourtool/diKTat/workflows/Build%20and%20test/badge.svg)
+![deteKT static analysis](https://github.com/saveourtool/diKTat/workflows/Run%20deteKT/badge.svg)
+![diKTat code style](https://github.com/saveourtool/diKTat/workflows/Run%20diKTat%20from%20release%20version/badge.svg?branch=master)
+[![codecov](https://codecov.io/gh/saveourtool/diKTat/branch/master/graph/badge.svg)](https://codecov.io/gh/saveourtool/diKTat)
 
-[![Releases](https://img.shields.io/github/v/release/cqfn/diKTat)](https://github.com/cqfn/diKTat/releases)
+[![Releases](https://img.shields.io/github/v/release/saveourtool/diKTat)](https://github.com/saveourtool/diKTat/releases)
 [![Maven Central](https://img.shields.io/maven-central/v/org.cqfn.diktat/diktat-rules)](https://mvnrepository.com/artifact/org.cqfn.diktat)
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fcqfn%2FdiKTat.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fcqfn%2FdiKTat?ref=badge_shield)
```
Comment (nulls) [inline README.md:10]: @akuleshov7, looks like we need to change url on FOSSA too: `https://app.fossa.com/projects/git%2Bgithub.com%2Fsaveourtool%2FdiKTat?ref=badge_shield` -- doesn't work

**Review by nulls — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (unix-junkie) [reply README.md]: @nulls, this has been addressed with 49bbef290f19f352c46ba06eb75506cb1957ada5.

File: README.md (hunk 2 above)
Comment (unix-junkie) [reply README.md:10]: 👀

Comment (unix-junkie) [reply README.md]: Addressed with 12351973fe1794d830f7a33e9013dbaa87291b3f, but @akuleshov7 please take a look first.

File: README.md [hunk 3]
```diff
@@ -7,7 +7,7 @@
 
 [![Releases](https://img.shields.io/github/v/release/saveourtool/diKTat)](https://github.com/saveourtool/diKTat/releases)
 [![Maven Central](https://img.shields.io/maven-central/v/org.cqfn.diktat/diktat-rules)](https://mvnrepository.com/artifact/org.cqfn.diktat)
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fcqfn%2FdiKTat.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fcqfn%2FdiKTat?ref=badge_shield)
```
Comment (unix-junkie) [inline README.md:10]: @akuleshov7 here

**Review by nulls — APPROVED** [review]
LGTM

**Pull request merged**